src/core_ct/importers.py

```diff
@@ -25,13 +30,30 @@ def dicom(dir: str = None, files: list[str] = None) -> Core:
     slices = []
     skipped: list[str] = []
     for f in files:
```

Comment:
So it turns out that the reason the `importers.dicom` function was failing for me is because it kept trying to import the `.DS_Store` that was also in the folder of scans. I know there is an option to ignore errors, but could you make it so it automatically ignores hidden files? I don't want to have to use the `force` option every time I import just to ignore the `.DS_Store` file and potentially miss out on any actual errors there are with DICOM files.

Reply:
I'll just add an option to ignore hidden files and default it to `True`, initially I had the `files` input to account for the .DS_STORE but that is cumbersome

Reply:
Added an option to ignore hidden files along with a regex search that should (hopefully) catch them

Reply:
Revised the code that checks for hidden files to use the `os` method `os.path.basename`